modules/tensor_mechanics/src/actions/TensorMechanicsAction.C

```diff
@@ -607,6 +655,63 @@ TensorMechanicsAction::actEigenstrainNames()
              << COLOR_DEFAULT;
 }
 
+void
+TensorMechanicsAction::verifyOrderAndFamilyOutputs()
+{
+  // Ensure material output order and family vectors are same size as generate output
+  auto order_check = getParam<MultiMooseEnum>("material_output_order");
+  auto family_check = getParam<MultiMooseEnum>("material_output_family");
+  bool order_display_notice = false;
+  bool family_display_notice = false;
+  // Magnitude check
+  if (order_check.size() > 1 && order_check.size() < _generate_output.size())
+    mooseError("The number of orders assigned to material outputs must be: 0 to be assigned "
+               "CONSTANT; 1 to assign all outputs the same value, or the same size as the number "
+               "of generate outputs listed.");
+  if (family_check.size() > 1 && family_check.size() < _generate_output.size())
+    mooseError("The number of families assigned to material outputs must be: 0 to be assigned "
+               "MONOMIAL; 1 to assign all outputs the same value, or the same size as the number "
+               "of generate outputs listed.");
+  // Make sure all outputs are standard constant value
+  if (order_check.size() == 0)
+  {
+    order_display_notice = true;
+    for (unsigned int i = 0; i < _generate_output.size(); ++i)
+      _material_output_order.push_back("CONSTANT");
+  }
+  if (family_check.size() == 0)
+  {
+    family_display_notice = true;
+    for (unsigned int i = 0; i < _generate_output.size(); ++i)
+      _material_output_family.push_back("MONOMIAL");
+  }
+
+  // For only one order, make all orders the same magnitude
+  if (order_check.size() == 1)
+  {
+    order_display_notice = true;
+    for (unsigned int i = 0; i < _generate_output.size(); ++i)
+      _material_output_order.push_back(_material_output_order[0]);
```

Comment:
did my suggestion not work?

Reply:
I tried it a couple of times.  I must of needed something else

Reply:
`_material_output_order` is a vector of strings.  That's why it wouldn't work.

Reply:
```
_material_output_order.assign(_generate_output.size(), _material_output_order[0]));
```